api/commands/compile.ts

```diff
@@ -55,8 +55,17 @@ export async function compile(
 
 export class CompileChildProcess {
   public static forkProcess() {
-    // Run the bin_loader script if inside bazel, otherwise don't.
-    const forkScript = process.env.BAZEL_TARGET ? "../vm/compile_loader" : "../vm/compile";
+    // Runs the worker_bundle script we generate for the package (see packages/@dataform/cli/BUILD)
+    // if it exists, otherwise run the bazel compile loader target.
+    const findForkScript = () => {
+      try {
+        const workerBundlePath = require.resolve("./worker_bundle");
+        return workerBundlePath;
+      } catch (e) {
```

Comment:
This is what I tried originally, unfortunately I need to apply the node resolution logic from require.resolve, and the only signal it gives is to throw or not to throw. Otherwise I have to start replicating that method, which makes me feel more uncomfortable than this.